app/components/node/PoSSize.tsx

```diff
@@ -142,6 +152,28 @@ const PoSSize = ({
     setNumUnit(commitments[index].numUnits);
   };
 
+  const validate = (value: number) => {
+    if (value < DEFAULT_POS_MAX_FILE_SIZE || Number.isNaN(value)) {
+      return DEFAULT_POS_MAX_FILE_SIZE;
+    }
+    if (value > DEFAULT_POS_MAX_FILE_SIZE * 50) {
```

Comment:
Such multiplication begins to be spread all around the code.
Let's encapsulate it in another constant:
```
export const DEFAULT_POS_MAX_FILE_SIZE = 1024 * 1024 * 1024 * 2; // 2GB
export const DEFAULT_POS_MAX_FILE_SIZE_LIMIT = DEFAULT_POS_MAX_FILE_SIZE * 50; // 100 GB
```
It will make our lives easier :)